scripts/mysql_system_tables_fix.sql

```diff
@@ -1045,6 +1045,55 @@ INSERT IGNORE INTO mysql.global_grants VALUES ('mysql.session', 'localhost', 'SY
 
 FLUSH PRIVILEGES;
 
+#
+# SQL commands to create the PXC SST root and role that is used by the SST process
```

Comment:
Different user.  This is the user that is used to create the sst user.  The security context for the server-side code is run with this user (this session then creates the SST user).

(Can't depend on the 'root' user existing as it can be renamed and can't depend on the anonymous user as that account could be locked down).

So the mysql.pxc.sst.root creates the mysql.pxc.sst.user (and will assign the mysql.pxc.sst.role when possible).  mysql.pxc.sst.root has the minimal set of privileges needed to create the mysql.pxc.sst.user.